swift/extractor/infra/SwiftDispatcher.h

```diff
@@ -265,7 +265,8 @@ class SwiftDispatcher {
  private:
   bool isLazyDeclaration(const swift::Decl& decl) {
     swift::ModuleDecl* module = decl.getModuleContext();
-    return module->isBuiltinModule() || module->getName().str() == "__ObjC";
+    return module->isBuiltinModule() || module->getName().str() == "__ObjC" ||
+           module->isNonSwiftModule();
```

Comment:
Do you know if the `__ObjC` module is also non-swift? If that's the case, we can remove the `__ObjC` specific test

Reply:
It is a Swift module, so we cannot remove it